yogstation/code/game/gamemodes/gangs/gangtool.dm

```diff
@@ -0,0 +1,260 @@
+	for(var/i in subtypesof(/datum/gang_item))
+		var/datum/gang_item/G = i
+		var/id = initial(G.id)
+		var/cat = initial(G.category)
+		if(id)
+			if(!islist(buyable_items[cat]))
+				buyable_items[cat] = list()
+			buyable_items[cat][id] = new G
+	.=..()
+
+/obj/item/device/gangtool/Destroy()
+	if(gang)
+		gang.gangtools -= src
+	return ..()
+
+/obj/item/device/gangtool/attack_self(mob/user)
+	..()
+	if (!can_use(user))
+		return
+	var/datum/antagonist/gang/boss/L = user.mind.has_antag_datum(/datum/antagonist/gang/boss)
+	var/dat
+	if(!gang)
+		dat += "This device is not registered.<br><br>"
+		if(L)
+			if(promotable && L.gang.leaders.len < L.gang.max_leaders)
+				dat += "Give this device to another member of your organization to use to promote them to Lieutenant.<br><br>"
+				dat += "If this is meant as a spare device for yourself:<br>"
+			dat += "<a href='?src=[REF(src)];register=1'>Register Device as Spare</a><br>"
+		else if(promotable)
+			var/datum/antagonist/gang/sweet = user.mind.has_antag_datum(/datum/antagonist/gang)
+			if(sweet.gang.leaders.len < sweet.gang.max_leaders)
+				dat += "You have been selected for a promotion!<br>"
+				dat += "<a href='?src=[REF(src)];register=1'>Accept Promotion</a><br>"
+			else
+				dat += "No promotions available: All positions filled.<br>"
+		else
+			dat += "This device is not authorized to promote.<br>"
+	else
+		if(gang.domination_time != NOT_DOMINATING)
+			dat += "<center><font color='red'>Takeover In Progress:<br><B>[DisplayTimeText(gang.domination_time_remaining() * 10)] remain</B></font></center>"
+
+		dat += "Registration: <B>[gang.name] Gang Boss</B><br>"
+		dat += "Organization Size: <B>[gang.members.len]</B> | Station Control: <B>[gang.territories.len] territories under control.</B> | Influence: <B>[gang.influence]</B><br>"
+		dat += "Time until Influence grows: <B>[time2text(gang.next_point_time - world.time, "mm:ss")]</B><br>"
+		dat += "<a href='?src=[REF(src)];commute=1'>Send message to Gang</a><br>"
+		dat += "<a href='?src=[REF(src)];recall=1'>Recall shuttle</a><br>"
+		dat += "<hr>"
+		for(var/cat in buyable_items)
+			dat += "<b>[cat]</b><br>"
+			for(var/id in buyable_items[cat])
+				var/datum/gang_item/G = buyable_items[cat][id]
+				if(!G.can_see(user, gang, src))
+					continue
+
+				var/cost = G.get_cost_display(user, gang, src)
+				if(cost)
+					dat += cost + " "
+
+				var/toAdd = G.get_name_display(user, gang, src)
+				if(G.can_buy(user, gang, src))
+					toAdd = "<a href='?src=[REF(src)];purchase=1;id=[id];cat=[cat]'>[toAdd]</a>"
+				dat += toAdd
+				var/extra = G.get_extra_info(user, gang, src)
+				if(extra)
+					dat += "<br><i>[extra]</i>"
+				dat += "<br>"
+			dat += "<br>"
+
+	dat += "<a href='?src=[REF(src)];choice=refresh'>Refresh</a><br>"
+
+	var/datum/browser/popup = new(user, "gangtool", "Welcome to GangTool v4.0", 340, 625)
+	popup.set_content(dat)
+	popup.open()
+
+/obj/item/device/gangtool/Topic(href, href_list)
+	if(!can_use(usr))
```

Comment:
Make 100% sure that this does not block crippled people from using it.

Reply:
yep works without legs